packages/terra-collapsible-menu-view/src/terra-dev-site/test/collapsible-menu-view/SingleItemGroup.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CollapsibleMenuView from '../../../CollapsibleMenuView';
+
+const locales = ['en', 'en-AU', 'en-US', 'en-GB', 'es', 'es-US', 'es-ES', 'de', 'fi-FI', 'fr', 'fr-FR', 'nl', 'nl-BE', 'pt', 'pt-BR', 'sv', 'sv-SE', 'fr-FR', 'nl', 'nl-BE', 'pt', 'pt-BR', 'sv', 'sv-SE'];
```

Comment:
`'fr-FR', 'nl', 'nl-BE', 'pt', 'pt-BR', 'sv', 'sv-SE'` is repeated twice, might throw a warning for `key={locale}` at line 14.